attractions/time-picker/time-picker.svelte

```diff
@@ -21,6 +21,10 @@
   export let value = null;
   export let format = '%H:%M';
   export let amPmTabName = 'am-pm';
+  export let hoursLabel = 'Hours';
```

Comment:
Wondering if these things should be slots. What do you think?

Reply:
Well this solution meets my needs but I have no objection to using slots.

Reply:
Because I believe that in some other cases we use named slots for such customization + it yields more freedom

Reply:
yeah, I get  that. Did you think to include the label as well in the <slot> or just the text?

Reply:
Sorry for the delay, I think it's best to include the label element in the slot for proper customization